mods/ts/rules/palettes.yaml

```diff
@@ -149,6 +141,18 @@
 	PlayerColorPalette:
 		BasePalette: player
 		RemapIndex: 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31
+	PaletteFromPaletteWithAlpha@placebuilding:
+		Name: placefootprint
+		BasePalette: ra
+		Alpha: 0.85
```

Comment:
I think this should be a bit lower , around 0.7, so that you still see the preview (test with invalid footprint of plugs, you hardly see the preview) and what is below an invalid line build footprint.

Reply:
Done.